src/AV/Video.cpp

```diff
@@ -1367,10 +1368,10 @@ int Video::addTimer(int delay)
 
 bool Video::addPacket(PacketQueue *queue, AVPacket *packet)
 {
-    av_dup_packet(packet);
+    AVPacket *dup = av_packet_clone(packet);
```

Comment:
Yes indeed, my bad, @verpeteren asked me and I overlooked the code, using `av_packet_ref()` is indeed the way to go.